src/Models/WebMapServiceCatalogGroup.js

```diff
@@ -59,6 +59,12 @@ var WebMapServiceCatalogGroup = function(application) {
      */
     this.blacklist = undefined;
 
+    /**
+     * Specifies whether to use each WMS layer's "title" (default), "name", or "abstract" as the primary title.
+     * @type {String}
+     */
+    this.titlefield = 'title';
+
     knockout.track(this, ['url', 'dataCustodian', 'parameters', 'blacklist']);
```

Comment:
thanks, I was hoping someone would tell me whether or not I needed to do that. should all fields be observable? any reason to not?